scripts/token-hud-extension.js

```diff
@@ -136,6 +140,49 @@ class TokenHudExtension {
 
    }
 
+   
+   static async addArmorTokenTip(app, html, data, actor) {
+
+        if (game.scenes.active.isView === false)
```

Comment:
Previously I would only show the Initiative tooltip on active scenes, but since 0.6 an optional setting has been added to choose whether to always show Initiative in the Hud Extension. Feels like the same should apply to Armour, or just to default show it always anyway. tbh I'm inclined to make that the standard behaviour as Settings are getting congested. Any other reason you are only wanting to show Armour on active scenes?